### PR TITLE
Rename Wasm.fsproj to Analysis.fsproj; add README

### LINKED ISSUE #4069: Add/improve docs around BlazorWorker+Wasm.fsproj

### LINKED ISSUE #4116: Rename Wasm.fsproj to Analysis.fsproj
What do you think, @pbiggar? We refer to Analysis pretty often, and that's all we (currently) use Wasm code for.

### PR DESCRIPTION
**Pull request by StachuDotNet**
This will resolve #4116 and resolve #4069.

### CODE REVIEW

File: fsharp-backend/paket.lock [hunk 1]
```diff
@@ -818,33 +818,33 @@ NUGET
...
       System.Drawing.Common (>= 5.0)
 
-GROUP Wasm
+GROUP Analysis
 NUGET
   remote: https://api.nuget.org/v3/index.json
-    Microsoft.AspNetCore.Authorization (6.0.1) - restriction: >= net6.0
-      Microsoft.AspNetCore.Metadata (>= 6.0.1) - restriction: || (>= net461) (>= netstandard2.0)
-      Microsoft.Extensions.Logging.Abstractions (>= 6.0) - restriction: || (>= net461) (>= netstandard2.0)
+    Microsoft.AspNetCore.Authorization (6.0.5) - restriction: >= net6.0
```
Comment (pbiggar) [inline fsharp-backend/paket.lock:824]: There's a lot of version changes here. How come and what should we do about them?

**Review by pbiggar — CHANGES_REQUESTED** [review]
Looks good. One open question before we can merge though.

File: fsharp-backend/paket.lock (hunk 1 above)
Comment (StachuDotNet) [reply fsharp-backend/paket.lock:824]: I think paket just refreshed some versions when I changed the group name from `wasm` to `analysis`.

Personally I wasn't concerned about them - they should be non-breaking changes, and we should have tests covering these things.

Comment (pbiggar) [reply fsharp-backend/paket.lock:824]: Cool, thanks for clarifying.

**Pull request merged**